### PR TITLE
Feature/create post

### PR DESCRIPTION
**Pull request by DmytroKolisnyk2**

### CODE REVIEW

File: packages/backend/src/repositories/product.ts [hunk 1]
```diff
@@ -131,32 +131,46 @@ export class ProductRepository {
...
+        minimalBid: data.minimalBid,
+        recommendedPrice: data.recommendedPrice,
+        endDate: data.endDate,
+        postDate: data.postDate,
       },
     });
   }
 
   public async update(id: string, data) {
+    console.log('dfsfdsdfs ', data.postDate);
```
Comment (YevheniiBieliaiev) [inline packages/backend/src/repositories/product.ts:155]: don't forget to delete :)

File: packages/frontend/validation-schemas/post/create-auction.ts [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    category: Joi.string()
+      .required()
+      .messages({
+        'string.empty': t('create-post:validation.category.empty'),
+      }),
+    currency: Joi.any().allow(''),
+    recommendedPriceCurrency: Joi.any().allow(''),
+    minimalBidCurrency: Joi.any().allow(''),
+    callingCode: Joi.any().allow(''),
+    site: Joi.any().allow(''),
```
Comment (YevheniiBieliaiev) [inline packages/frontend/validation-schemas/post/create-auction.ts:16]: in QA rules we have type URL, so i recommend add something like that
facebook: Joi.string()
      .allow('')
      .uri()
      .max(ValidationRanges.MAX_SOCIAL_NETWORK_URI_SYMBOLS)
      .messages({
        'string.uri': 'error message',
        'string.max': 'error message',
      }),

File: packages/frontend/validation-schemas/post/create-auction.ts [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+      .max(50)
+      .messages({
+        'string.max': t('create-post:validation.title.long'),
+        'string.empty': t('create-post:validation.title.empty'),
+      }),
+
+    description: Joi.string()
+      .trim()
+      .required()
+      .max(6000)
```
Comment (YevheniiBieliaiev) [inline packages/frontend/validation-schemas/post/create-auction.ts:38]: not too large? :))

Comment (YevheniiBieliaiev) [reply packages/frontend/validation-schemas/post/create-auction.ts:38]: oh, ok. sorry!)

File: packages/backend/src/repositories/product.ts [hunk 4]
```diff
@@ -137,17 +137,23 @@ export class ProductRepository {
...
```
Comment (Nightlngale) [inline packages/backend/src/repositories/product.ts]: console.log

**Review by Nightlngale — APPROVED** [review]

**Review by YevheniiBieliaiev — APPROVED** [review]

**Review by dannleed — APPROVED** [review]

**Pull request merged**